Jenkinsfile

```diff
@@ -68,7 +68,7 @@ pipeline {
                     steps {
                         runTest script: '''set -ex
                                            rm -f *.xml
-                                           echo "<failure bla bla bla/>" > \
+                                           echo "<failure> bla bla bla</failure>" > \
```

Comment:
Shouldn't this test be expanded to test for both types of failure reporting now that we know it can be in multiple formats?

Reply:
I considered that, but given that the older test had not been failing, just covering the newly discovered test was good enough instead of adding a new test stage.